CHANGELOG.md

```diff
@@ -1,5 +1,160 @@
 # Changelog
 
+## v6.1.0-preview.1 - 2018-03-23
+
+### Breaking Changes
+
+- Throw terminating error in `New-TemporaryFile` and make it not rely on the presence of the `TEMP` environment variable (#6182) (Thanks @bergmeister!)
+- Remove unsupported members from the enum 'Language' in `Add-Type` (#5829) (Thanks @iSazonov!)
+- Fix range operator to work better with character ranges (#5732) (Thanks @iSazonov!)
+
+### Engine Updates and Fixes
+
+- Fix `ValidateSet` with generator in a module (#5702)
+- Update `SAL` annotation and fix warnings (#5617)
+- Add `ForEach` and `Where` methods to `PSCustomobject` (#5756) (Thanks @iSazonov!)
+- Add `Count` and `Length` properties to `PSCustomobject` (#5745) (Thanks @iSazonov!)
+- Make minor fixes in compiler to properly handle void type expression (#5764)
+- Logging: Fix the escaped characters when generating `.resx` file from PowerShell `ETW` manifest. (#5892)
+- Remove `PSv2` only code from `Types_Ps1Xml.cs` and `HostUtilities.cs` (#5907) (Thanks @iSazonov!)
+- Do not pass `ArgumentAst` to `CreateArgument` for `pwsh -encodedarguments` (#5836)
+- Logging: Handle path that contains spaces in `RegisterManifest.ps1` (#5859) (Thanks @tandasat!)
+- Add `-settingsfile` to `pwsh` to support loading a custom powershell config file. (#5920)
+- Return better error for `pwsh -WindowStyle` on unsupported platforms. (#5975) (Thanks @thezim!)
+- Enable conversions from `PSMethod` to `Delegate` (#5287) (Thanks @powercode!)
+- Minor code clean-up changes in tab completion code (#5737) (Thanks @kwkam!)
+- Add lambda support to `-replace` operator (#6029) (Thanks @IISResetMe!)
+- Fix error in windows provider when the environment as an existing set of variables name that only differs by case (#6320)
```

Comment:
Title is confusion.  Use: Fix retrieval of environment variables on Windows in cases where variable names differ only by case.